sympy/concrete/tests/test_sums_products.py

```diff
@@ -252,6 +252,8 @@ def test_geometric_sums():
     # issue 8251:
     assert summation((1/(n + 1)**2)*n**2, (n, 0, oo)) == oo
 
+    #issue 9908:
+    assert Sum(1/(n**3 - 1),(n,-oo,-2)).doit() == summation(1/(n**3 - 1),(n,-oo,-2))
```

Comment:
I had say that the current test is legitimate. Since if in the future the `Sum(1/(n**3 - 1), (n, -oo, -2)).doit()` is made to be evaluated, then it would not equal `Sum(1/(n**3 - 1), (n, -oo, -2))` (the un-evaluated expression).